README.md

```diff
@@ -350,8 +350,10 @@ overriding methods then please test your work before upgrading.
 
 ## Reporting Issues
 
-Please [create an issue](https://github.com/dnadesign/silverstripe-elemental/issues) for any bugs you've found, or features you're missing.
+Please [create an issue](https://github.com/silverstripe/silverstripe-elemental/issues) for any bugs you've found, or features you're missing.
 
 ## Credits
 
+Silverstripe Elemental was created by [DNA Design](https://www.dna.co.nz/).
```

Comment:
Not sure if this is sufficiently deferential. I can make it more grandiose if need be.